src/w2toolbar.js

```diff
@@ -143,7 +143,7 @@
                 $(this.box).find('#tb_'+ this.name +'_item_'+ w2utils.escapeId(it.id)).remove();
                 // remove from array
                 var ind = this.get(it.id, true);
-                if (ind != null) this.items.splice(ind, 1);
+                if (ind >= 0) this.items.splice(ind, 1);
```

Comment:
I understand that 0 is a valid index. My point is 
```js
if (0 != null) - returns true
if (0 >= 0) - returns true
```
So, there is no difference what to use. Check in browser console for yourself.